setup.py

```diff
@@ -106,7 +106,6 @@
         'Topic :: Internet :: WWW/HTTP :: Dynamic Content',
         'Topic :: Software Development :: Libraries :: Python Modules',
         'Programming Language :: Python :: 3',
-        'Programming Language :: Python :: 3.5',
         'Programming Language :: Python :: 3.6',
         'Development Status :: 1 - Planning',
```

Comment:
```suggestion
        'Development Status :: 3 - Alpha',
```

I guess it will be released as alpha?